scripts/install.sh

```
@@ -184,6 +189,6 @@ install_skills
 echo "Installed $app_dest"
 echo "CLI: $bin_dest"
 echo "Config dir: $profile_dir/"
-echo "Apps: $(ls "$profile_dir/apps" | wc -l | tr -d ' ') synced from examples/"
+echo "Apps: $(ls "$profile_dir/apps" | wc -l | tr -d ' ') synced from apps/"
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)